spec/models/note_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Note, type: :model do
-  subject { build(:note) }
+  subject { build(:note, notable_type: "Student") }
```

Comment:
@Luke-kb if you going to build a note, it probable good to make it realistic and build a student first as well
so you have an actual notable to refer to.
you could do something like this?
```
let(:student) { build(:student) }
subject { build(:note, notable: student) }
```
But a cleaner way is to use [Factories with associations](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#associations).
Then just building a note will auto create the noteable object as well.
```subject { build(:note) } ```

Then later on when you add more Noteable objects, you can add [traits](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#traits)
```
FactoryBot.define do
  factory :note do
    title { Faker::Lorem.sentence }
    content { (Faker::Lorem.paragraphs(number: 2, supplemental: true)).join(" ") }
    noteable { create(:student) }
  end
end
```